stable/vpa/templates/recommender-podmonitor.yaml

```
@@ -0,0 +1,26 @@
+{{- if and .Values.recommender.enabled .Values.recommender.podMonitor.enabled }}
```

I believe wrapping these files in this if statement will fix the e2e tests for now until chart-testing allows us to skip that check.

```suggestion
{{- if .Values.recommender.podMonitor }}
{{- if and .Values.recommender.enabled .Values.recommender.podMonitor.enabled }}
```